plugins/org.eclipse.osee.ats.core/src/org/eclipse/osee/ats/core/version/AtsVersionServiceImpl.java

```diff
@@ -327,6 +327,16 @@ public Collection<IAtsVersion> getVersions(IAtsTeamDefinition teamDef) {
    public Version createVersion(String title, IAtsChangeSet changes) {
       return createVersion(title, Lib.generateArtifactIdAsInt(), changes);
    }
+   
+   @Override
+   public Version createVersion(String name, String description, ArtifactId teamId) {
+      IAtsChangeSet changes = atsApi.createChangeSet("Create Version: "+name);
+      Version newVersion = createVersion(name,changes);
+      newVersion.setDescription(description);
```

Comment:
I'd load the teamId and make sure it exists and isOfType AtsArtifactTypes.TeamDefinition.  Exception if it doesn't.